test/executor.test.js

```diff
@@ -906,6 +906,43 @@ describe('executor', function() {
     });
   });
 
+  describe.only('when booting with lazy connect', function() {
```

Comment:
Forgotten `only`, please remove. BTW if you want to run a subset of tests only, you can use mocha's command-line flag `-g`: 

```bash
$ mocha -g "lazy connect"
```

That way you don't need to change code.